.github/workflows/pr.yml

```diff
@@ -130,3 +130,44 @@ jobs:
         env:
           # needed for github-action-config.json generation
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  # Note: the command `run` is tested by the previous steps (`make test`).
+  commands:
```

Comment:
how to not forget to update this :D 

maybe we need bash magic like

1. run `golangci-lint 42`
2. parse `Available Commands:`
3. loop: run command from 2, parse `Available Commands` again
4. etc

But, of course, explicit is better than implicit 🤷‍♂️